examples/pnnl-example1.md

```diff
@@ -0,0 +1,26 @@
+# PNNL Example Building 1
+
+This is an example building model provided by Pacific Northwest National Laboratory.
+
+The model is a simplistic, reference building consisting of a single story with 5 rooms: open office, private office, bathroom, kitchenette, corridor.
+
+The model contains a representation of the building architecture and lighting system.
+
+In addition to the s223 ontology, the model uses the [Real Estate Core ontology](https://dev.realestatecore.io/ontology/) to describe space types.
+
+## Source
+
+The model was created from souce data consisting of a Revit building information model (BIM) exported via a [Speckle](https://speckle.systems/) interface.
+
+### The Speckle view of rooms and lighting fixtures
+<img width="1212" alt="image" src="https://github.com/open223/models.open223.info/assets/22898727/92b1afd2-b3c3-492b-a7c7-61dea827a246">
+
+Model instance data have a @prefix bdg1: <http://speckle.xyz/streams/1fed8e620e/objects/> that is resolvable to a Speckle URL.
+
+For example, bdg1:05749166d93671bedf16efb52636ce38 a s223:Luminaire resolves to https://speckle.xyz/streams/1fed8e620e/objects/05749166d93671bedf16efb52636ce38.
```

Comment:
```suggestion
For example, the following resolves to https://speckle.xyz/streams/1fed8e620e/objects/05749166d93671bedf16efb52636ce38.
```ttl
bdg1:05749166d93671bedf16efb52636ce38 a s223:Luminaire
```
```

Reply:
Matt, it is a version of Pritoni. I will add some mention of that after talking to Steve Ray so we can sync on the sentence that we will both use when describing the origin of this "reference" building. I'm thinking we might not want to just call it the Pritoni building, but rather reference the Metadata schemas journal article as its origin.

Reply:
Ok, sounds good. It would be nice to use the same file name prefix as @steveraysteveray's Pritoni model. I'm assuming this one is more electrical focused and Steve's is mechanical? If that's the case, maybe `<new-name>_elec.ttl` or similar.

Reply:
@steveraysteveray's model is one of the two Pritoni models that were used to explore and demonstrate new concepts, as they were developed by Strike Team Bravo. Parastoo, Steve, and I developed a Pritoni model using TopBraid (referred to as the PSM Pritoni model). Joel and Christian developed a Pritoni model using Bob. At various times the concepts the two models contained were similar, but they were not strictly synchronized. 

The PSM Pritoni model contains architecture, mechanical, and lighting/electrical. I think it contains more concepts than any other validated model, at this point.

Our (PNNL)'s tool development has been driven by an attempt to develop a Pritoni model that progresses, over time, to include all the concepts that the PSM Pritoni model includes. We are not there yet, but we  aim to get there.

All of this to say, I don't think it makes sense to differentiate the file names by what they represent. The fundamental difference is the development method (TopBraid vs. Revit-based workflow) and the development team (PSM vs. PNNL).

I assume that NREL and LBNL will develop additional example models beyond those currently posted? How are you thinking you will differentiate them? I initially just attempted to match your file naming convention (nrel-example, lbnl-example) and then differentiate by adding a number (example1, example2). There is likely a better way. We will soon have a model for LBNL Bldg 59 and NREL Rail Bldg, and later in the year a model for LBNL Bldg 33. So, there will be more instances of multiple models for the same building.

How about we agree on a numbering convention for each building, and then append the development team to that building number in the file name? The development method can be described on the model page, as it is now.

For example, 

Pritoni = bdg1, filenames = bdg1-psm.ttl, bdg1-pnnl.ttl 

Proto Medium Office = bdg2, filenames = bdg2-pnnl.ttl, bdg2-nrel.ttl (if you end up creating a Building Motif version of this model)

LBNL Bldg 59 = bdg3, filenames = bdg3-pnnl.ttl, bdg3-lbnl.ttl

Reply:
Ok, sounds like we should rename files later after agreeing on a convention and more are added.

Reply:
Here's some food for thought from the QUDT world. Each of the QUDT graphs contains a reference to metadata information from the ontology instance, such as:

```
<http://qudt.org/2.1/vocab/unit>  vaem:hasGraphMetadata vaem:GMD_QUDT-UNITS-ALL ;
```

and here's what we store. We might want to cherry-pick some of these concepts, rather than trying to encode it all in the file name. (Having said that, we do use a file naming convention as well to distinguish schema files from vocabulary files, hence VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```
vaem:GMD_QUDT-UNITS-ALL
  a vaem:GraphMetaData ;
  dcterms:contributor "Jack Hodges" ;
  dcterms:contributor "Simon J D Cox" ;
  dcterms:contributor "Steve Ray" ;
  dcterms:created "2019-07-30"^^xsd:date ;
  dcterms:creator "Steve Ray" ;
  dcterms:modified "2023-11-15T10:34:16.143-05:00"^^xsd:dateTime ;
  dcterms:rights """
  This product includes all or a portion of the UCUM table, UCUM codes, and UCUM definitions or is derived from it, subject to a license from Regenstrief Institute, Inc. and The UCUM Organization. Your use of the UCUM table, UCUM codes, UCUM definitions also is subject to this license, a copy of which is available at ​http://unitsofmeasure.org. The current complete UCUM table, UCUM Specification are available for download at ​http://unitsofmeasure.org. The UCUM table and UCUM codes are copyright © 1995-2009, Regenstrief Institute, Inc. and the Unified Codes for Units of Measures (UCUM) Organization. All rights reserved.

THE UCUM TABLE (IN ALL FORMATS), UCUM DEFINITIONS, AND SPECIFICATION ARE PROVIDED 'AS IS.' ANY EXPRESS OR IMPLIED WARRANTIES ARE DISCLAIMED, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
  """ ;
  dcterms:rights "The QUDT Ontologies are issued under a Creative Commons Attribution 4.0 International License (CC BY 4.0), available at https://creativecommons.org/licenses/by/4.0/. Attribution should be made to QUDT.org" ;
  dcterms:subject "Units-All" ;
  dcterms:title "QUDT Units Version 2.1 Vocabulary" ;
  vaem:description "Standard units of measure for all units." ;
  vaem:graphName "qudt" ;
  vaem:graphTitle "QUDT Units Version 2.1.33" ;
  vaem:hasGraphRole vaem:VocabularyGraph ;
  vaem:hasOwner <http://www.linkedmodel.org/schema/vaem#QUDT.org> ;
  vaem:hasSteward <http://www.linkedmodel.org/schema/vaem#QUDT.org> ;
  vaem:intent "To provide a vocabulary of all units." ;
  vaem:isMetadataFor <http://qudt.org/2.1/vocab/unit> ;
  vaem:latestPublishedVersion "https://qudt.org/doc/2023/11/DOC_VOCAB-UNITS-ALL-v2.1.html"^^xsd:anyURI ;
  vaem:logo "https://qudt.org/linkedmodels.org/assets/lib/lm/images/logos/qudt_logo-300x110.png"^^xsd:anyURI ;
  vaem:namespace "http://qudt.org/vocab/unit/"^^xsd:anyURI ;
  vaem:namespacePrefix "unit" ;
  vaem:owner "QUDT.org" ;
  vaem:previousPublishedVersion "https://qudt.org/doc/2023/10/DOC_VOCAB-UNITS-ALL-v2.1.html"^^xsd:anyURI ;
  vaem:revision "2.1" ;
  vaem:turtleFileURL "http://qudt.org/2.1/vocab/unit"^^xsd:anyURI ;
  vaem:usesNonImportedResource dcterms:abstract ;
  vaem:usesNonImportedResource dcterms:created ;
  vaem:usesNonImportedResource dcterms:creator ;
  vaem:usesNonImportedResource dcterms:modified ;
  vaem:usesNonImportedResource dcterms:title ;
  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
  rdfs:label "QUDT Unit of Measure Vocabulary Metadata Version 2.1.33" ;
.
```

Reply:
Christian and I are continuing to work on the CJ model for Pritoni built using Bob (and Wendy, Lofty, and Dizzy) and I'll have a Cornell small office building.  I hope there are lots more models to add!

Reply:
@MatthewSteen @gtfierro it sounds like you are okay with the current status of my branch; I will go ahead and merge tonight unless I hear from you. I will also create an Issue to start discussing a new file naming convention, and move my proposal from here to the issue.

@JoelBender good to know. Yes I'm hoping for lots more models too 👍 

@steveraysteveray I'm down to experiment with using dcterms and vaem ontologies to add metadata to the shared models. Maybe you and I can develop a minimum proposal for differentiating between the Pritoni models (PSM, CJ, PNNL) next week, and then see what the group thinks?